src/Nordea/Components/Coachmark.elm

```diff
@@ -0,0 +1,329 @@
+module Nordea.Components.Coachmark exposing
+    ( OptionalConfig(..)
+    , page
+    , view
+    )
+
+import Css
+    exposing
+        ( absolute
+        , alignItems
+        , auto
+        , backgroundColor
+        , before
+        , borderRadius
+        , borderStyle
+        , bottom
+        , cursor
+        , displayFlex
+        , fixed
+        , flexStart
+        , height
+        , int
+        , justifyContent
+        , left
+        , marginLeft
+        , none
+        , num
+        , opacity
+        , padding
+        , padding2
+        , pct
+        , pointer
+        , position
+        , relative
+        , rem
+        , right
+        , spaceBetween
+        , top
+        , transform
+        , translate2
+        , transparent
+        , width
+        , zIndex
+        )
+import Css.Global
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes as Attrs exposing (css)
+import Html.Styled.Events as Events
+import List.Extra as List
+import Maybe.Extra as Maybe
+import Nordea.Components.Button as Button
+import Nordea.Components.Text as Text
+import Nordea.Components.Tooltip as Tooltip
+import Nordea.Html as Html
+import Nordea.Html.Attributes as Attrs
+import Nordea.Resources.Colors as Colors
+import Nordea.Resources.I18N exposing (Translation)
+import Nordea.Resources.Icons as Icons
+import Nordea.Themes as Themes
+import Set
+
+
+type OptionalConfig
+    = HighlightedClass String
+    | Placement Tooltip.Placement
+    | ShowPage (Maybe Int)
```

Comment:
Does this depict the step number of the "coachmark tutorial"? I associate `page` with a full size page. Could `step` be a bit more accurate? 🤔

Reply:
Agreed, changed it to step